PopUp-Window Project

AJ193 opened the pull request:
- Generate work section from js file
- Create the popup section
- style the popup section

AJ193 committed:
modify html,css file.

Esmatullah12 committed:
Update index.html file

Modify the index.html file.

codepantha commented on app.js:
```diff
@@ -1,24 +1,207 @@
+const popup = document.createElement('div');
+function createPopup(nbr) {
+  popup.innerHTML = `
+  <div id="popup-container" class="popup-window">
+  <div class="popup-holder">
+    <div class="popup-close-btn-holder">
+      <div class="title-xbtn">
+        <h1 id="popup-header" class="project-heading project-heading-pop">${projects[nbr].name}</h1>
+        <svg
+          id="x-popup"
+          class=".xBtn"
+          xmlns="http://www.w3.org/2000/svg"
+          width="32"
+          height="32"
+          viewBox="0 0 24 24"
```
- [x] Going by the Figma design, please ensure to give some space on the left and right of the popup so that the blurred background image can be more visible just like the Figma design
## Figma design
![image](https://user-images.githubusercontent.com/46839548/231908213-3f56afde-e0b1-47e0-adb5-cba39e00c764.png)
## Your Design
![image](https://user-images.githubusercontent.com/46839548/231908258-726b7385-0604-486e-a122-13946ade36e5.png)

codepantha reviewed (changes requested):
# STATUS: CHANGES REQUIRED ♻️ 

Hi @AJ193 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🎯 
✔️ Descriptive PR
✔️ Popup modal 📌 
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AJ193 committed:
Modify css file

As Described by the reviewer i have made some changes in the code to match the figma design

Olamarx reviewed (approved):
Hi @AJ193, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
<img src="https://media4.giphy.com/media/mPIA4KZVXv0ty/giphy.gif"/>
## Highlights
- Linters passed 👍🏼
- Descriptive PR ✔️ 
- Pop up functionality ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
